var/spack/repos/builtin/packages/attr/package.py

```diff
@@ -12,6 +12,11 @@ class Attr(AutotoolsPackage):
     homepage = "https://savannah.nongnu.org/projects/attr"
     url = "http://download.savannah.gnu.org/releases/attr/attr-2.4.47.src.tar.gz"
 
+    maintainers("cosmicexplorer")
+
+    # FIXME: spack does not validate any of the checksums for this package for some reason!
```

Comment:
```
> spack checksum --verify attr 2.4.48 2.4.47 2.4.46
```

All seem to work for me. What do you mean by Spack not validating the checksums?